tests/std/tests/P1208R6_source_location/test.cpp

```diff
@@ -241,7 +244,9 @@ constexpr void lambda_test() {
 #elif defined(__clang__) // ^^^ basic / detailed Clang vvv
     assert(fun2 == "auto " THISCALL_OR_CDECL " lambda_test()::(anonymous class)::operator()(void) const"sv);
 #elif defined(__EDG__) // ^^^ detailed Clang / detailed __EDG__ vvv
-    assert(fun2 == "lambda []()->auto::operator()()->auto"sv);
+    // TRANSITION, EDG is changing to resemble C1XX's output somewhat more closely

```

Comment:
nit - this "TRANSITION" comment is different from the ones above.
The variants I see are:
* EDG is changing to resemble C1XX's output somewhat more closely
* EDG is changing to almost match C1XX's output
* EDG is changing to match C1XX's output 

Were these supposed to be the same (for easy CTRL+F'ing) or is their difference meaningful?

Reply:
The difference is meaningful. For lambdas, the output differs radically, but EDG is at least adding calling conventions.

For the "almost match" ones, EDG is still missing "struct" before "std::source_location" for reasons that are unclear to me.